### PR TITLE
Use proper index for bulk size evaluation

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
With this commit we specify an index template and use a matching index
in all bulk-size evaluation challenges. Due to restrictions with the
maximum number of clients we also reduce the number of concurrent
clients from 200 to 100 in the concurrency challenge.

### CODE REVIEW

File: eventdata/challenges/bulk-size-evaluation.json [hunk 1]
```diff
@@ -114,89 +130,53 @@
...
         }
       }
     },
     {
-      "name": "index-append-10-200clients",
+      "name": "index-append-{{concurrency_evaluation_bulk_size}}-{{concurrency_evaluation_clients}}clients",
       "operation": {
         "operation-type": "bulk",
         "param-source": "elasticlogs_bulk",
-        "bulk-size": 10,
```
Comment (dliappis) [inline eventdata/challenges/bulk-size-evaluation.json]: AFAIU the j2 loop above will use [1, 10, 50] and since we set `set concurrency_evaluation_bulk_size = 1000 // factor` this will result in bulk sizes of 1000, 100 and **20**; the old setting in this line uses bulk-size **10**; is this change on purpose?

Comment (danielmitterdorfer) [reply eventdata/challenges/bulk-size-evaluation.json]: Yes, this is indeed on purpose because (at least on my machine) handling 100 clients was ok whereas 200 was not (it led to connection drops in the actor system)

Comment (dliappis) [reply eventdata/challenges/bulk-size-evaluation.json]: Thanks SGTM

**Comment by danielmitterdorfer** [comment]
Thanks for the review!
